config.lua

```diff
@@ -3,7 +3,7 @@ Config = {}
 -- NPC Vehicle Lock States
 Config.LockNPCDrivingCars = true -- Lock state for NPC cars being driven by NPCs [true = locked, false = unlocked]
 Config.LockNPCParkedCars = true -- Lock state for NPC parked cars [true = locked, false = unlocked]
-
+Config.UseKeyfob = false -- you can set this true if you dont need ui
```

Comment:
It might be an idea to have this at the top of the config file as it's pretty global here

A name like `Config.Enabled = false` may be a little more descriptive, too?

Reply:
Hi, thanks for your suggestion
But I don't think it will increase anything ❤️